sdk/iot/README.md

```diff
@@ -1,4 +1,4 @@
-# Embedded C for Azure IoT Clients
+# Azure IoT Clients
 
 Azure SDK for Embedded C official IoT Hub client library.

```

Comment:
Is this accurate?

Reply:
on line 3? I believe so. "Azure SDK for Embedded C" is the top level name so what follows seems reasonable

Reply:
Yes, I meant the other part `official IoT Hub client library.`

The title is `Azure IoT Clients`, but the tagline mentions `Hub`. Is that intentional?

This seems similar to the one in sdk/iot/hub/readme.md
`"Official Embedded C client for Azure IoT Hub.`

Reply:
Good catch @ahsonkhan .  That is wrong: we have two separate services: Device Provisioning Service and IoT Hub. They are not the same and we shouldn't mix things up. The /iot folder will contain all IoT related clients.

Reply:
Oh, I take that back: It is called Azure IoT Hub Device Provisioning Service.